enhancements/kubelet/oci-image-volume.md

```diff
@@ -0,0 +1,187 @@
+---
+title: oci-image-volume
+authors:
+  - bitoku
+reviewers:
+  - haircommander
+  - saschagrunert
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - mrunalp
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: 2025-05-07
+last-updated: 2025-05-07
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPNODE-2929
+  - https://issues.redhat.com/browse/OCPNODE-2575
+see-also:
+  - N/A
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+# OCI Image Volume
+
+## Summary
+
+In Kubernetes 1.33, support for OCI Image Volume went to Beta.
+This enhancement is to enable this feature in OpenShift.
+See [KEP-4639: Summary](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4639-oci-volume-source#summary)
+
+## Motivation
+
+See [KEP-4639: Motivation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4639-oci-volume-source#motivation)
+
+### User Stories
+
+See [KEP-4639: User Stories](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4639-oci-volume-source#user-stories-optional)
+
+### Goals
+
+- Enable ImageVolume feature gate for OpenShift.
+
+### Non-Goals
+
+- This proposal does not aim to replace existing VolumeSource types.
+- This proposal does not address other use cases for OCI objects beyond directory sharing among containers in a pod.
+- Mounting thousands of images in a single pod.
```

Comment:
I kinda find it odd this is here tbh.are hundreds supported? seems odd to set such an arbitrarily high limit with no testing of those limits

Reply:
I might have copied from the original KEP. https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4639-oci-volume-source#non-goals
Do we want to just delete this item or completely replace this section to the link to the upstream KEP?

Reply:
Deleted the line.